content/docs/1.13/apis.md

```diff
@@ -0,0 +1,81 @@
+---
+title: APIs
+hasparent: true
+---
+
+Jaeger components implement various APIs for saving or retrieving trace data.
+
+TODO
+
+  * stable
+  * restricted
+  * deprecated
+
+## Span reporting APIs
+
+Agent and Collector are the two components of the Jaeger backend that can receive spans and forward them to storage.
+
+### Thrift over UDP (stable)
```

Comment:
Could you also document which reporting mechanisms are available on agent or collector?

It is mostly written in the text but maybe adding list of components with this port/format would be more readable.

Reply:
I was thinking if we need to have a matrix showing which component supports which APIs. Although at this point I think those are completely disjoined sets